src/containerapp/azext_containerapp/_params.py

```diff
@@ -334,3 +339,34 @@ def load_arguments(self, _):
         c.argument('environment_name', options_list=['--environment'], help="The environment name.")
         c.argument('resource_group_name', arg_type=resource_group_name_type, id_part=None)
         c.argument('configuration', nargs="*", help="Java component configuration. Configuration must be in format \"<propertyName>=<value> <propertyName>=<value> ...\".")
+
+    with self.argument_context('containerapp sessionpool') as c:
+        c.argument('name', options_list=['--name', '-n'], help="The Session Pool name.")
+        c.argument('resource_group_name', arg_type=resource_group_name_type, id_part=None)
+
+    with self.argument_context('containerapp sessionpool', arg_group='Configuration') as c:
+        c.argument('managed_env', validator=validate_managed_env_name_or_id, options_list=['--environment'], help="Name or resource ID of the container app's environment.")
```

Comment:
why add managed env in Configuration?

Reply:
Session pool has 2 types, JupyterPython do not have environment, CustomerContainer need to provide managed environment as parameter, just like containerapps do.

Reply:
So it is managed_env is not a require parameter. But if you are saying in other parameter group, I can it to Container group.